src/zshrc

```diff
@@ -170,6 +170,15 @@ bindkey -M menuselect '^o' accept-and-infer-next-history
 bindkey '^[[A' history-substring-search-up
 bindkey '^[[B' history-substring-search-down
 
+# Use the provided Bash autocompletion for adr-tools. Note that there are other
+# scripts in this directory, but not all of them work well with zsh.
+autoload -U +X bashcompinit && bashcompinit
+source /opt/homebrew/etc/bash_completion.d/adr-tools
+
+# Some tools (e.g. Git, adr-tools) use $VISUAL and $EDITOR as a fallback.
+# Traditionally, $VISUAL is intended to be a fully-featured editor, and $EDITOR
+# is set to a line editor (e.g. ed), but modern tools often ignore $VISUAL.
 export EDITOR=nano
+export VISUAL=nano
```

Comment:
Is this a useful change here? I feel like it's probably a separate thing to enabling autocomplete, right, given things fall back to `EDITOR` if `VISUAL` isn't set.

Reply:
OK, maybe let's lose this as it's probably not something many people will want to reconfigure.

Reply:
Remember, this is intended to be a set of defaults that the user of this script then maintains themself. If they want this distinction, they can bring it in for themself.